stock_picking_eco_tag/models/stock_move_line.py

```diff
@@ -0,0 +1,18 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    def lines_category_product(self):
+
+        for attribute in self.product_id.attribute_line_ids:
+            #if attribute.id = self.env.ref("stock_picking_eco_tag.product_attribute_category").id
+            if attribute.attribute_id.name == "CATEGORÍA":
+                if attribute.value_ids.name:
+                    return attribute.value_ids.name
+
+        return False
```

Comment:
Is it possible to optimize this code like hre: https://github.com/solvosci/slv-stock/blob/13.0/stock_picking_deliveryslip_subtotals/models/stock_picking.py#L10 ?

If we have a picking with a lot of lines with the same product, the code becomes inefficient

And please protect code, because it will fail if that attribute has more than a value